### PR TITLE
Some new code

### PR DESCRIPTION
**Pull request by SergeyMezentsev**
This is the code for PR from VS Code

### CODE REVIEW

**Review by SergeyMezentsev — COMMENTED** [review]
Here we can see comments done from the VS Code!!!

File: app/main.cpp [hunk 1]
```diff
@@ -3,7 +3,11 @@
 
 int main(int argc, char* argv[]) {
```
Comment (SergeyMezentsev) [inline app/main.cpp:5]: Delete first cout!

File: app/main.cpp [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
 
-    std::cout << "Hello, World!" << std:: endl;
+    std::cout << "1 - Hello, World!" << std:: endl;
+    std::cout << "2 - Hello, World!" << std:: endl;
+    std::cout << "3 - Hello, World!" << std:: endl;
+    std::cout << "4 - Hello, World!" << std:: endl;
+    // @Serjuice3
```
Comment (Serjuice3) [inline app/main.cpp]: Please delete your username here!

File: app/main.cpp (hunk 1 above)
Comment (Serjuice3) [inline app/main.cpp:5]: Delete first cout from the file

**Review by Serjuice3 — COMMENTED** [review]
Here you have something to work on!

File: app/main.cpp (hunk 1 above)
Comment (SergeyMezentsev) [reply app/main.cpp:5]: Done!

File: app/main.cpp (hunk 2 above)
Comment (SergeyMezentsev) [reply app/main.cpp]: Done!

**Review by SergeyMezentsev — COMMENTED** [review]
Ok, everything's done!

**Comment by Serjuice3** [comment]
Wow, that's great!

**Pull request merged**